appveyor.yml

```diff
@@ -1,7 +1,7 @@
 branches:
   only:
     - master
-    - develop
+    - fix-win-install-message
```

Comment:
oops, just noticed this, please remove branch, add back `develop`

Reply:
yup will do once i i see cypress-bot post the right thing